src/core/monitor/gnss_synchro_monitor.cc

```diff
@@ -91,6 +92,7 @@ int gnss_synchro_monitor::general_work(int noutput_items __attribute__((unused))
                         }
                 }
         }
+    consume(d_nchannels - 1, ninput_items[d_nchannels - 1]);
```

Comment:
again, the logic is not obvious. Why is channel `N-1` not subject to decimation? Or any processing?

Reply:
Same thing.
I'll add comment from observables block before this line.